src/lib/components/EventSchedule/index.js

```diff
@@ -82,6 +93,13 @@ class EventSchedule extends HTMLElement {
       el.removeAttribute('tabindex');
     });
 
+    // If the user opens this page from externally on a specific session, make sure we're showing
+    // the correct day of tab.
+    const index = this._tabsElement.indexOfTabParent(session);
+    if (index !== -1) {
+      this._tabsElement.focusTab(index);
```

Comment:
I think you may not want this because it's literally calling focus() on the tab when the page loads (note the blue ring). So if someone opens the page with a screen reader it'll immediately announce "Day 1, tab"

![image](https://user-images.githubusercontent.com/1066253/85086599-e9d96680-b18f-11ea-92c5-24679ab6515f.png)

Reply:
oh it also scrolls the page down so we definitely don't want that.

Reply:
Good catch. I now just set `.activeTab`.